src/Components/Address/utils/useInitialLocationValues.ts

```diff
@@ -25,6 +25,22 @@ export const useInitialLocationValues = (
       return {}
     }
 
+    // If no countryInputOptions provided, only return phoneNumberCountryCode
+    if (!countryInputOptions) {
+      const matchingPhoneCountry = countryPhoneOptions.find(
+        country =>
+          location.country &&
+          (country.value.toLowerCase() === location.country.toLowerCase() ||
```

Comment:
Instead of comparing with `location.country` which is the full name, maybe we need `location.countryCode` which is the alpha-2 code? We might need to add `countryCode` to the query fields.